conanfile.py

```diff
@@ -1,47 +1,87 @@
 import os
 
-from conans import ConanFile, CMake, tools
-from os import path
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.env import VirtualRunEnv
+from conan.tools.files import load
 
 
 class LibCosimCConan(ConanFile):
+    # Basic package info
     name = "libcosimc"
+
+    def set_version(self):
+        self.version = load(self, os.path.join(self.recipe_folder, "version.txt")).strip()
+
+    # Metadata
+    license = "MPL-2.0"
     author = "osp"
-    exports = "version.txt"
-    scm = {
-        "type": "git",
-        "url": "auto",
-        "revision": "auto"
-    }
+    description = "A C wrapper for libcosim, a co-simulation library for C++"
+
+    # Binary configuration
+    package_type = "library"
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "virtualrunenv"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.options["*"].shared = self.options.shared
+
+    # Dependencies/requirements
+    tool_requires = (
+        "cmake/[>=3.15]",
+        "doxygen/[>=1.8]",
+    )
     requires = (
-        "libcosim/0.10.2@osp/stable"
-        )
+        "libcosim/0.10.3@osp/testing-bugfix_transitive-libs-boost",
```

Comment:
Maybe this have been changed to `libcosim/0.10.3@osp/stable` before merging.

Reply:
Oops! I've pushed a hotfix to `master` now.

Reply:
Looks like my hotfix broke master. :( I guess the issue is that the upstream fix in open-simulation-platform/libcosim#753 is in libcosim `master`, but not in any release yet, so it's not in the `stable` channel. I don't know what the right way to fix this is. Revert all changes to libcosimc `master` and reopen this PR again, or make libcosimc `master` depend on the libcosim `testing` channel which tracks the `master` branch?